performance/projects/01_mini_project_dont_merge/dbt_project.yml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
The name of this file makes me thing you don't want this in the repo...is that correct?

Reply:
That's correct. I'll replace it with one with like 2000 files which will take an eternity to run. But not while I'm developing 🙃